src/Microsoft.SqlTools.ServiceLayer/SchemaCompare/SchemaCompareOperation.cs

```diff
@@ -120,8 +120,9 @@ public void Execute(TaskExecutionMode mode)
                     }
                 }
 
-                // Appending the set of errors that are stopping the schema compare to the ErrorMessage 
-                var errorsList = ComparisonResult.GetErrors().Select(e => e.Message).Distinct().ToList();
+                // Appending the set of errors that are stopping the schema compare to the ErrorMessage
+                // GetErrors return all type of warnings, and error messages. Only filtering the error type messages here
+                var errorsList = ComparisonResult.GetErrors().Where(x => x.MessageType.Equals(Microsoft.SqlServer.Dac.DacMessageType.Error)).Select(e => e.Message).Distinct().ToList();
                 ErrorMessage = string.Join("\n", errorsList);
```

Comment:
just to triple confirm - the ErrorMessage doesnt get passed to ADS if the result is true (passe) - right?

Reply:
True, if the comparison is success, the ErrorMessage will be empty!